<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selector</class>
 <widget class="QMainWindow" name="Selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лавка</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_drink">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>147</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Напитки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_vegetable">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>103</y>
      <width>74</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Овощи</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>23</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_fruit">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>59</y>
      <width>78</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Фрукты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>370</y>
      <width>70</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cart">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>370</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Корзина</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Молоко</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_3">
       <property name="text">
        <string>110</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Манго</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_8">
       <property name="text">
        <string>400</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Вода</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_10">
       <property name="text">
        <string>45</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Бананы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_12">
       <property name="text">
        <string>90</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Картофель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_2">
       <property name="text">
        <string>60</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Лук</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_9">
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Огурцы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_11">
       <property name="text">
        <string>135</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Кофе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_14">
       <property name="text">
        <string>800</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Персики</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_4">
       <property name="text">
        <string>120</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Чай</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_6">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Редис</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_15">
       <property name="text">
        <string>80</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Яблоки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_1">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Баклажаны</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_7">
       <property name="text">
        <string>150</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Морковь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_5">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,1,1">
     <item>
      <widget class="ClickableLabel" name="lbl_item_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Мандарины</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_cost_13">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_q_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>Selector</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
